Automattic/simplenote-macos
# Decoupling ShareViewController

### Details:
- Splits `ShareViewController` UI from the MainNib
- Reimplements `ShareViewController` in Swift

@diegoreymendez sir, may I bug you with this one?
Thanks in advance!

Ref. #458 

### Test
1. Launch Simplenote and open any note
2. Press the top right (i) button
3. Press over `Collaborate`

- [x] Verify the ShareViewController shows correctly
- [x] Verify Share looks great in Dark Mode

### Release
These changes do not require release notes.
